DotNetRu.Android/Helpers/LocaleUtils.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.Content.Res;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using Java.Util;
+
+namespace DotNetRu.Droid.Helpers
+{
+    public class LocaleUtils
+    {
+
+        private static Locale sLocale;
```

Comment:
Is it just a local variable to store current locale? If yes, please rename it to currentLocale.
BTW, why it's static?